backend/src/services/release.ts

```diff
@@ -276,11 +282,36 @@ export async function getReleasesForExport(user: UserInterface, modelId: string,
   return releases
 }
 
+export function semverStringToObject(semver: string) {
```

Comment:
We can give this an explicit return type to ensure it's not broken by accident:
```
export function semverStringToObject(semver: string): SemverObject { ...
```

Reply:
Added